packages/block-editor/src/components/media-replace-flow/test/index.js

```diff
@@ -57,11 +70,21 @@ describe( 'General media replace flow', () => {
 				name: 'Replace',
 			} )
 		);
-
 		const uploadMenu = screen.getByRole( 'menu' );
 
 		expect( uploadMenu ).toBeInTheDocument();
 		expect( uploadMenu ).not.toBeVisible();
+
+		/**
+		 * The popover will be displayed and positioned with a slight delay,
```

Comment:
The comment is not 100% correct because the popover will be displayed immediately, only the positioning is delayed.

Reply:
And I would reorder the assertions: first check that the popover is positioned, and only then start asserting on the `uploadMenu` element.

The test will succeed anyway, but the reordered test tells a better story to a human reader: 1) open the popover 2) wait for it to be displayed and positioned and whatever other dust needs to settle 3) check stuff in the popover.

Reply:
Thanks for the hints! I've improved the comment in 99069f9877c01318a60dd10428408f28fb0265a0 and reordered assertions in 8032ab04b126b2e874479e1cb2ff8507f16e6fe0.